### ISSUE TITLE
Similar genome finder will create group with bad genome

### ISSUE THREAD

**Issue by olsonanl** [PATRIC3/patric3_website#2254]
User created a group with the output of similar genome finder; it included a bad genome that then was suppressed from any views of the genome group, but was still present in the group so that a CodonTree invocation on that group crashed.

Tagged for similar genome finder, but this may be a more general problem.

Related to https://jira.cels.anl.gov/browse/PATRIC-2426

**Comment by mshukla1** [comment]
This is most likely because the genome was deleted from the database due to bad quality, but corresponding mash files are not deleted. Need top delete such genomes from ftp/blast/mash dbs.

I will clean up ftp and blast files, can you do the same for mash sketches?

**Comment by olsonanl** [comment]
Yes. We should also have the services verify the existence of genomes before returning them to the user; perhaps also think about verifying genome groups before their use. This will happen again and users might have genomes in their groups that have been deleted.
